simapp/simd/cmd/genaccounts.go

```diff
@@ -40,30 +39,32 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 		Args: cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			clientCtx := client.GetClientContextFromCmd(cmd)
-			depCdc := clientCtx.Codec
-			cdc := depCdc.(codec.Codec)
 
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
 
 			config.SetRoot(clientCtx.HomeDir)
 
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			keyringBackend, _ := cmd.Flags().GetString(flags.FlagKeyringBackend)
+
+			var kr keyring.Keyring
```

Comment:
let's move the 4 lines above into the `if` statement - they are not used outside.